pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/ElasticTestUtil.java

```diff
@@ -234,13 +236,56 @@ public static Profile addParticipantProfileFromFile(String esIndex, String fileN
         }
     }
 
+    public static Dsm addParticipantDsmFromFile(String esIndex, String fileName, String ddpParticipantId, String dob) {
```

Comment:
Since this is a test utility, it's okay to just have the method declare `throws Exception`.  If we want to avoid throwing the exception, consider narrowing the scope of the `try` so that it only captures the checked exception, like so:

```
Gson gson = new Gson();
        String json = null;
        try {
            json = TestUtil.readFile(fileName);
        } catch (Exception e) {
            e.printStackTrace();
            Assert.fail("Could not read test json for participant " + ddpParticipantId);
            return null;
        }
 ```

Putting too much stuff into the `try` block tends to result in exception handling that is hard to debug.  If we narrow it here, we can make a more specific error message, instead of a generic "oops, something went wrong".  We know exactly what went wrong--we could read the json file.